addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2388,6 +2388,8 @@ export function enforceWhitespace(el, offset, direction, rule) {
             } else if (isVisibleStr(node)) {
                 break;
             }
+        } else {
+            break;
```

Comment:
Thanks for this fix ! 👍 You can now write a proper commit message for it 😉

Reply:
Let me know if you like it :)

Reply:
I like it very much, good job ! 👍 A few remarks though:
- You should mention that the "code sections" or "code blocks" you are talking about are PRE nodes, otherwise it's not necessarily obvious for an external reader what you call "code blocks". Just adding "(pre)" somewhere would be enough.
- I feel like the "Environment" part is supposed to provide steps to reproduce the issue but it does so in a confusing way IMO. I mean, you wrote nice sentences and everything, and honestly it looks good, but it's harder for people in the future to "reconstruct" somewhat the actual steps from the sentences. I sugest you re-use the steps to reproduce from the task description, I remember them being easier to follow.

Reply:
Thanks ! I made the steps more similar to the task spec but still a bit more generic (and without specifying the copy-paste issue since it's a corner case and got fixed alongside the basic case, it' doesn't really add value to describe it).

Reply:
I thought I remembered something with the text "ab" something like that, but I checked on the task and I couldn't find it anymore so I guess I mixed it up with another task. Thanks !

Reply:
It is on the task ! It just really doesn't matter that it's 'ABC\nDEF' or anything else, just that it's at least two lines and the last line is at least two characters.

Reply:
Modified in https://github.com/odoo/odoo/pull/118899

Reply:
Keeping the two (generic explanation + concrete example) is even better. Thank you ! 👍